docs/user-guide/configuration/annotations.md

```diff
@@ -35,6 +35,7 @@ Some Annotations are used to modify the properties of the build environment. The
 | beta.screwdriver.cd/executor | Ask your cluster admin | This will determine what compute system is used to run the build. For example, set the build to run in a VM, a kubernetes pod, a docker container, or a Jenkins agent. |
 | beta.screwdriver.cd/cpu | `MICRO` / `LOW` / `HIGH` | When using a `k8s` executor, this will allow the user to choose between 0.5 (`MICRO`), 2 (`LOW`) and 6 (`HIGH`) CPU resources. Default is `LOW`.<br>   When using a `k8s-vm` executor, this will allow the user to choose between 1 (`MICRO`), 2 (`LOW`) and 6 (`HIGH`) CPU resources. Default is `LOW`. |
 | beta.screwdriver.cd/ram | `MICRO` / `LOW` / `HIGH` | When using a `k8s` executor, this will allow the user to choose between 1 GB (`MICRO`), 2 GB (`LOW`) and 12 GB (`HIGH`) RAM. Default is `LOW`.<br> When using a `k8s-vm` executor, this will allow the user to choose between 1 GB (`MICRO`), 2 GB (`LOW`) and 12 GB (`HIGH`) RAM. Default is `LOW`. |
+| beta.screwdriver.cd/disk | `LOW` / `HIGH` | When using a `k8s-vm` executor, this will allow the user to choose between 20 GB (`LOW`) and 50 GB (`HIGH`) disk space. Default is `LOW`. |
```

Comment:
Not sure if this makes more or less sense, but should this be alphabetically positioned in relation to the other resource annotations? (beta.screwdriver.cd/cpu, beta.screwdriver.cd/disk, beta.screwdriver.cd/ram)

Reply:
sounds reasonable!